src/sentry/api/validators/broadcast.py

```diff
@@ -12,3 +12,5 @@ class AdminBroadcastValidator(BroadcastValidator):
     isActive = serializers.BooleanField(required=False)
     dateExpires = serializers.DateTimeField(required=False, allow_null=True)
     cta = serializers.CharField(max_length=256, required=True)
+    mediaUrl = serializers.URLField(required=False)
```

Comment:
title should be updated to 64 in this validator too

Reply:
👍 good catch, changed